report-api/src/test/java/no/sikt/nva/data/report/api/fetch/FetchNviInstitutionReportHandlerProxyTest.java

```diff
@@ -91,4 +159,24 @@ private static String getExpectedProblem(String requestId) {
                                                                  .with("requestId", requestId)
                                                                  .build())).orElseThrow();
     }
+
+    private void mockResponse(String responseBody, String contentType)
+        throws IOException, InterruptedException {
+        var response = mockHttpResponse(responseBody, contentType);
+        mockResponse(response);
+    }
+
+    private void mockResponse(HttpResponse<String> response)
+        throws IOException, InterruptedException {
+        when(authorizedBackendClient.send(any(), any(BodyHandler.class))).thenReturn(response);
```

Comment:
Struggled to match the expected request here. Initially tried this:
```
var expectedRequest = generateExpectedRequest(...);
when(authorizedBackendClient.send(eq(expectedRequest), any(BodyHandler.class))).thenReturn(response);
```
Not a fan of using `any()` as argument matcher..